libvast/vast/chunk.hpp

```diff
@@ -180,10 +180,16 @@ class chunk final : public caf::ref_counted {
       };
       deleter_ = std::move(g);
     } else {
-      deleter_ = std::move(step);
+      deleter_ = std::forward<Step>(step);
```

Comment:
Note to the reviewer: While not strictly related to the PR, I noticed that we wrongly used move rather than forward on a forwarding reference here.